plugins/reports/counter/classes/releases/CounterRelease3_0.inc.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file plugins/generic/sushiLite/classes/releases/CounterRelease3_0.inc.php
+ *
+ * Copyright (c) 2014 University of Pittsburgh
+ * Distributed under the GNU GPL v2 or later. For full terms see the file docs/COPYING.
+ *
+ * @class CounterRelease3_0
+ * @ingroup plugins_reports_counter
+ *
+ * @brief A COUNTER report, release 3.0
+ */
+
+import('plugins.reports.counter.classes.releases.CounterRelease');
+
+class CounterRelease3_0 extends CounterRelease {
```

Comment:
As I mentioned above, if you think you will need to use composition (implement separated release classes and let the reports use them) instead of inheritance (implement only report classes that implement themselves their releases details) than that's ok. I can't decide too much because I don't see all the requirements, but any approach we take must be implemented in a clear way to avoid confusion.

Looking at the code right now, I think we don't need to use composition. Just implement a general report class that also contains all the release interface that a report needs, than a more specific one that implements an specific release detail, and then the reports of that specific releases can all inherit from that. 

It would be something like: CounterReport, Release3CounterReport, JR1CounterReport, JR2CounterReport, etc. The two last ones inheriting from the second one, that inherits from the first one. Does that makes sense?